common/ReduxChannel.js

```diff
@@ -0,0 +1,87 @@
+const DEFAULT_OPTIONS = {
```

Comment:
This is just moved over from `content-src`, with one change (had to check existence of `window` so it doesn't throw)